src/examples/rest/src/frontend/types.ts

```diff
@@ -21,179 +21,316 @@ export type Scalars = {
   JSON: { input: any; output: any; }
 };
 
-export type AdminUiEntityAttributeMetadata = {
-  __typename?: 'AdminUiEntityAttributeMetadata';
-  isReadOnly?: Maybe<Scalars['Boolean']['output']>;
+export type Query = {
```

Comment:
I'd love to make this file `types.generated.ts` just so it matches what we're doing on the backend.

Reply:
I was just keeping the same name, but I had the exact same question. I'll change it.